firebase-appdistribution/src/main/res/layout/activity_feedback.xml

```diff
@@ -1,86 +1,139 @@
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_weight="0"
+        android:background="?android:attr/listDivider" />
+
+    <TextView
+        android:id="@+id/senderText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/sender"
+        android:textSize="24sp"
+        android:padding="16dp" />
+
+    <View
+        android:id="@+id/divider1"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_weight="0"
+        android:background="?android:attr/listDivider" />
+
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_weight="3"
+        android:padding="16dp" >
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical" >
+
+            <EditText
+                android:id="@+id/feedbackText"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:layout_weight="3"
+                android:autofillHints=""
+                android:background="@null"
+                android:gravity="start"
+                android:hint="@string/feedback_text_hint"
+                android:inputType="textMultiLine"
+                android:minLines="5"
+                android:singleLine="false"
+                android:textAlignment="textStart"
+                android:paddingBottom="24dp"
+                tools:ignore="InefficientWeight,TextContrastCheck" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:background="@drawable/card_background"
+                android:orientation="vertical">
+
+                <CheckBox
+                    android:id="@+id/screenshotCheckBox"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:text="@string/screenshot" />
+
+                <ImageView
+                    android:id="@+id/screenshotImageView"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:scaleType="fitXY"
+                    android:contentDescription="@string/screenshot_image_description" />
+            </LinearLayout>
+        </LinearLayout>
+    </ScrollView>
+
+    <View
+        android:id="@+id/divider2"
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:background="?android:attr/listDivider" />
+
+    <TextView
+        android:id="@+id/infoText"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text=""
```

Comment:
Is this necessary or could we remove it?

Reply:
Done